core/src/backend/audio/decoders.rs

```diff
@@ -235,9 +236,6 @@ pub trait SeekableDecoder: Decoder {
     }
 }
 
-/// `StreamTagReader` reads through the SWF tag data of a `MovieClip`, extracting
-/// audio data from the `SoundStreamBlock` tags. It can be used as an `Iterator` that
-/// will return consecutive slices of the underlying audio data.
```

Comment:
This deduplication confused me terribly... turns out, this doc stutter has been lurking in here for three years now... :sweat_smile: